src/core/rootSaga.js

```diff
@@ -1,6 +1,7 @@
 import { all } from "redux-saga/effects";
 import { popularMoviesSaga } from "../features/MoviesList/popularMoviesSaga";
+import { GenresSaga } from "../features/MoviesList/Genres/genresSaga";
 
 export default function* rootSaga() {
-  yield all([popularMoviesSaga()]);
+  yield all([popularMoviesSaga(), GenresSaga()]);
```

Comment:
I think this way will looks better:
yield all([
popularMoviesSaga(),
genresSaga()]);

also generesSaga (there is big G letter instead of g)

Reply:
I've made changes.